### ISSUE TITLE
AngularHTTP

### ISSUE THREAD

**Issue by ghaiat** [Kinvey/js-library#3]
Hello
You recently changed the request method in kinvey-angular AngularHTTP object .
(I am using kinvey-angular 1.1.13 and angular v1.3.12)

It use to be :
```js
return $http({
data: body,
headers: headers,
method: method,
timeout: options.timeout,
url: url
}).then(function(response) {
```

but now in the latest version it is :
```js
return $http({
data: body,
headers: headers,
method: method,
timeout: options.timeout,
url: url
}).then(function(response, status, headers) {
```
and the headers object is null so the line 9001:
```js
var responseContentType=headers('Content-Type')||undefined;
```
fails which make all request from kinvey not working.

**Comment by ThomasConner** [comment]
Hey @ghaiat. We have noted the bug and are working on a fix. What version of angular are you using?

**Comment by ghaiat** [comment]
Hello
angular v1.3.12

**Comment by ThomasConner** [comment]
Just released 1.1.14 for the AngularJS library. Please update to this release. Thanks.
